mission.html

```diff
@@ -1,114 +1,129 @@
-  </p>
-  <br>
-  <br>
-  <h3>What is Autism?</h3>
-  <p>Also called: autism spectrum disorder, is a serious developmental disorder that impairs the ability to communicate and interact.
-    Autism spectrum disorder impacts the nervous system and affects the overall cognitive, emotional, social and physical health of the affected individual.
-    The range and severity of symptoms can vary widely. Common symptoms include difficulty with communication, difficulty with social interactions, obsessive interests and repetitive behaviours.
-    Early recognition, as well as behavioural, educational and family therapies may reduce symptoms and support development and learning.
-  </p>
-  <br>
-  <br>
-  <h3>What are its symptoms?</h3>
-  <p>
-    People may experience the following symptoms: <br>
+    <h3> How our game helps children and parents? </h3>
+    <p>Since most of the autistic children struggle with making decisions or choosing the right emotions.
+      So our game provides an platform to solve this issue with fun. ALso we are using google actions interactive
+      canvas for the quiz app
+      so that it will help the autistic children with language and communication skills along with emotion
+      recognition in daily life.
+      Moreover are helping parents to teach their autistic children in a good way so that they know more about their
+      child and help them with autism.
+    <p>
+    </p>
+    <br>
     <br>
-    Abnormal Body Posturing or Facial Expressions.<br>
-    Abnormal Tone of Voice.<br>
-    Avoidance of Eye Contact or Poor Eye Contact.<br>
-    Behavioral Disturbances.<br>
-    Deficits in Language Comprehension.<br>
-    Delay in Learning to Speak.<br>
-    Flat or Monotonous Speech.<br>
-    Inappropriate Social Interaction.<br>
+
+    <h3>What is Autism?</h3>
+    <p>Also called: autism spectrum disorder, is a serious developmental disorder that impairs the ability to
+      communicate and interact.
+      Autism spectrum disorder impacts the nervous system and affects the overall cognitive, emotional, social and
+      physical health of the affected individual.
+      The range and severity of symptoms can vary widely. Common symptoms include difficulty with communication,
+      difficulty with social interactions, obsessive interests and repetitive behaviours.
+      Early recognition, as well as behavioural, educational and family therapies may reduce symptoms and support
+      development and learning.
+    </p>
+    <br>
+    <br>
+    <h3>What are its symptoms?</h3>
+    <p>
+      People may experience the following symptoms: <br>
+      <br>
+      Abnormal Body Posturing or Facial Expressions.<br>
+      Abnormal Tone of Voice.<br>
+      Avoidance of Eye Contact or Poor Eye Contact.<br>
+      Behavioral Disturbances.<br>
+      Deficits in Language Comprehension.<br>
+      Delay in Learning to Speak.<br>
+      Flat or Monotonous Speech.<br>
+      Inappropriate Social Interaction.<br>
 
```

Comment:
remove these extra kines, it's repeated. till line 157

Reply:
I have reverted the PR so could you please make a new one with the changes requested?

Reply:
do i have to remove that using another pull request?